cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -99,6 +100,15 @@ def test_course_listing_is_escaped(self):
         self.assertEqual(response.status_code, 200)
         self.assert_no_xss(response, escaping_content)
 
+    def test_empty_course_listing(self):
+        """
+        Test on empty course listing, studio name is properly displayed
+        """
+        message = "Are you staff on an existing {studio_name} course?".format(studio_name=settings.STUDIO_SHORT_NAME)
```

Comment:
I think it would be `Studio` but not sure entirely. It is being used on all over the `index` template in studio.